test/TESTING.md

```diff
@@ -12,3 +12,8 @@ Should result in:
 
 - An image named `pihole:[branch-name]` being built
 - Tests being ran to confirm the image doesn't have any regressions
+
+# Modify Pipfile
+
+You can enter into the test docker image using `./build-and-test.sh enter`.
```

Comment:
👌 Did I accidentally remove this from the docs before? I vaguely recall "discovering" the functionality when refactoring things and thinking it was really useful! V good for debugging single tests, without having to re-run the whole script!

Reply:
I think it was always undocumented. After having learned it, forgot it, then relearned it- I thought it might be nice to document it.